### DISCUSSION TITLE
[BUG] Mica effect for File Explorer windows

### DISCUSSION THREAD

**Discussion by hiranokite** [valinet/ExplorerPatcher#1019] [General]
Version - 22000.493.42.30

```
.30

    Implemented Mica effect for File Explorer windows (thanks @MishaTY) and option to hide the icon and title of File Explorer windows
```

This option has no effect if  "Launch folder windows in separate process" in "Folder Options" is enabled

![ss](https://user-images.githubusercontent.com/5361459/156937786-d44e04e0-2520-456f-a06d-3cb0b39deea8.png)

**Comment by valinet** [comment]
Yeah, my bad, it will be fixed in the next pre-release, thanks for pointing it out.
